app/src/main/res/menu/activity_calendar_list.xml

```diff
@@ -17,6 +17,11 @@
           android:onClick="onToggleDarkMode"
           app:showAsAction="never" />
 
+    <item android:id="@+id/privacy_policy"
+          android:title="@string/calendar_list_privacy_policy"
+          android:onClick="onPrivacyPolicyRequested"
```

Comment:
Doesn't really matter, but I'd call it `onShowPrivacyPolicy` (always onShowXXX, because we already have that for `onShowInfo`).